### DISCUSSION TITLE
Black screen on non-root login on Proxmox

### DISCUSSION THREAD

**Discussion by TheGreenGuy5** [Ultra980/danora-linux#8] [General]
After rebasing a Kinoite install to Danora Linux (`ghcr.io/ultra980/danora:20230108` image), logging in with any user other than `root`, results in a black screen (only cursor is visible). The `sddm` login manager works just fine.

**Comment by TheGreenGuy5** [comment]
Same behaviour seems to carry over to the `rawhide` image

**Comment by gr4phite** [comment]
This didn't happen in Gnome Boxes (flatpak). Perhaps it's specific to proxmox?

**Comment by gr4phite** [comment]
> This didn't happen in Gnome Boxes (flatpak). 

I also used Silverblue, not Kinoite, although it shouldn't matter...

**Comment by gr4phite** [comment]
[This comment](https://discussion.fedoraproject.org/t/booting-silverblue-kinoite-using-podman-docker/44457/5) seems to show that there is at least one... Maybe there are more?

**Comment by gr4phite** [comment]
Users need to be able to test this before they install it.

**Comment by TheGreenGuy5** [comment]
I have installed Danora on a **Thinkpad T420**, and no issues appeared, so this seems to be a VM (Proxmox)-related problem. I am closing this issue as VM-related problems are of very low importance

**Comment by gr4phite** [comment]
It's still an issue, I think it's useful for users to test it in a VM.

**Comment by TheGreenGuy5** [comment]
This issue seems to only be affecting **Proxmox**, which uses VNC to connect to the VM. This might be a VNC-related issue.

**Comment by gr4phite** [comment]
Did you check in another software e.g. VirtualBox?

**Comment by TheGreenGuy5** [comment]
> Did you check in another software e.g. VirtualBox?

Yup, it works fine in Virtualbox. Might try to VNC into the VM or figure out how to not use VNC on Proxmox (to test if this is VNC-related).

**Comment by gr4phite** [comment]
Well, proxmox isn't meant for this sort of thing anyways. There's no point in making an immutable VM in Proxmox, since it's not the base system. Closing as wontfix since it doesn't help our goal.

**Comment by gr4phite** [comment]
Moved this to a discussion since we might fix this eventually. But we have more important stuff to do now.
